README.rst

```diff
@@ -10,27 +10,43 @@ Test Management Tool
 Description
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ``tmt`` python module and command-line tool implement the
-Metadata Specification which allows storing all needed test
-execution data directly within a git repository. In this way, it
-makes testing independent on any external test management system.
+The ``tmt`` tool provides a user-friendly way to work with tests.
+It allows to comfortably create new tests, safely and easily run
```

Comment:
I don't believe `allow to verb` is grammatically correct, it should be something like
`allows comfortably creating`
Source: https://english.stackexchange.com/questions/60271/grammatical-complements-for-allow/60285#60285